pydantic/env_settings.py

```diff
@@ -77,7 +78,7 @@ def _build_environ(
             if env_val is None:
                 continue
 
-            if field.is_complex():
+            if field.is_complex() and not lenient_issubclass(field.outer_type_, SplitStr):
```

Comment:
This is what lets Settings read the string as is without trying to parse JSON. And then the `SplitStr` (one of `CommaSeparated`, `SpaceSeparated`, or `CommaSeparatedStripped`) class parses the string, separates the values, and the rest of the validation continues (with internal types, etc).